server/testutils/factories/offenceDto.ts

```diff
@@ -0,0 +1,15 @@
+import { faker } from '@faker-js/faker/locale/en_GB'
+import { Factory } from 'fishery'
+
+import type { OffenceDto } from '@prison-api'
+
+export default Factory.define<Partial<OffenceDto>>(() => ({
```

Comment:
I'm curious how this works, e.g. in the client the expected return value is a `Promise<PageOffenceDto>` which has a `content` property which is an array of `OffenceDto`s (not partial). In the test this factory is used, but I wouldn't have thought these would typecheck as `OffenceDto`s given they miss some required keys 🤔

Might be a gap in my TypeScript knowledge here!